docs/deployments/domains.mdx

```diff
@@ -2,14 +2,68 @@
 title: Domains
 ---
 
-A domain is the address of your deployment.
+A domain is the address used to access your Globe-hosted project. By default, each project is assigned a unique domain (e.g., `your-project.globe.com`). You can also link a custom domain to your project for a branded experience.
 
-Each deployment, production or preview gets assigned a unique domain that directly points to the specific deployment. If the deployment is not yet ready or successful, visiting a domain will redirect you to the dashboard for that deployment.
+## Linking a Custom Domain
 
-## Production domains
+Custom domains allow you to connect your branded URL (e.g., `your-company.com`) to your Globe-hosted project. This makes your project accessible on a domain of your choice.
 
-Each project gets assigned a unique domain when it is created. Whenever you successfully deploy to the production environment, the production domain will be updated to point to the latest deployment.
+### How It Works
 
-### Custom Domains
+1. Register a domain with an external provider (e.g., Namecheap, GoDaddy, Google Domains).
+2. Configure your DNS settings to point to Globe.
+3. Verify ownership in the Globe Dashboard.
+4. Cloudflare handles DNS management and security, including automatic SSL (HTTPS).
 
-Custom domains are not yet supported. We are working on it.
+Currently, each Globe project can have only one custom domain. To add a new domain, you must remove the existing one first. We will be adding support for multiple domains soon.
+
+## Setting Up Your Custom Domain
+
+### Step 1: Configure DNS with Your Registrar
+
+Before adding a custom domain to Globe, update your DNS settings with your domain provider (e.g., GoDaddy, Google Domains, Cloudflare).
+
+- Add a **CNAME record** pointing your domain to the value provided by Globe.
```

Comment:
We should put the CNAME record in the docs - it should be static and never change. @kaziwaseef thoughts?